src/ops/ops.go

```diff
@@ -76,6 +78,7 @@ func NewOps(logger *logrus.Logger, proxySet bool) Ops {
 
 // ExecPrivilegeCommand execute a command in the host environment via nsenter
 func (o *ops) ExecPrivilegeCommand(liveLogger io.Writer, command string, args ...string) (string, error) {
+	o.log.Infof("Ececuting privilege command: %s %s", command, args)
```

Comment:
```suggestion
	o.log.Infof("Executing privileged command: %s %s", command, args)
```

Reply:
Just stayed consistent with the function name.

Reply:
Okay, but "Executing" is still spelled incorrectly :wink:

Reply:
lol, missed that, fixing, I was lazy to run spell-check.